controller/users_blackbox_test.go

```diff
@@ -50,6 +50,51 @@ func (s *TestUsersSuite) SecuredServiceAccountController(identity account.Identi
 	return svc, NewUsersController(svc, s.GormDB, s.Configuration)
 }
 
+func (s *TestUsersSuite) TestObfuscateUserAsServiceAccountBadRequest() {
+	// given
+	user := s.createRandomUser("TestObfuscateUserAsServiceAccountBadRequest")
+	identity := s.createRandomIdentity(user, account.KeycloakIDP)
+	secureService, secureController := s.SecuredServiceAccountController(identity)
+	// when
+	idAsString := "bad-uuid"
+	test.ObfuscateUsersBadRequest(s.T(), secureService.Context, secureService, secureController, idAsString)
+}
+
+func (s *TestUsersSuite) TestObfuscateUserAsServiceAccountOK() {
+	// given
+	user := s.createRandomUser("TestObfuscateUserAsServiceAccountOK")
+	identity := s.createRandomIdentity(user, account.KeycloakIDP)
+	// when
+	secureService, secureController := s.SecuredServiceAccountController(identity)
+
+	test.ObfuscateUsersOK(s.T(), secureService.Context, secureService, secureController, (user.ID).String())
```

Comment:
Some validation missing here :)
Please retrieve the user again, and ensure both the `identity` and the `user` entities have different field values than what they were before the obfuscation was done.

Reply:
Indeed, added in commit b5abde4